# Copyright 2019 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#    https://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

# General planner params
cycle_time: 0.5
reinitialization_distance: 1.0

# Parameters for the primitive planner
robot_radius: 0.3
obstacle_check_front_offset: 0.3
horizon: 50
horizon: 20
primitive_length: 0.5
primitive_min_turn_radius: 0.5
num_primitive_curvature_samples: 7
max_curvature_change_per_meter: 1.5
# Cost Weights and constriants
travel_cost_weight: 500
path_divergence_weight: 10
path_end_divergence_weight: 100
path_end_alignment_weight: 100
max_path_divergence: 2
obstacle_weight: 1
curvature_integral_weight: 1000
# Visited grid
visited_grid_spatial_resolution: 0.2
num_visited_grid_angle_bins: 64
visited_grid_curvature_resolution: 0.3

# Timeouts
clicked_path_timeout: 0.5
odometry_timeout: 0.2
perception_timeout: 0.5
